src/sentry/integrations/gitlab/client.py

```diff
@@ -26,6 +26,7 @@ class GitLabApiClientPath:
     project_issues = "/projects/{project}/issues"
     project_hooks = "/projects/{project}/hooks"
     project_hook = "/projects/{project}/hooks/{hook_id}"
+    project_search = "/projects/{project}/search"
```

Comment:
oop this was from the attempt on the search endpoint. might be good to explain what happened. I couldn't figure out exactly the syntax until I found https://gitlab.com/gitlab-org/gitlab/-/issues/25263.

This suggested that I would be able to do something like below, but this returned `[]`.
```
/search?scope=blobs&search=* filename:*codeowners
```
When I tried the following, I was able to get a result, but that was because I had `@megadee` in my file contents

```
/search?scope=blobs&search=@ filename:*codeowners
```